x-pack/plugins/security_solution/server/endpoint/lib/artifacts/lists.ts

```diff
@@ -261,7 +322,7 @@ function translateEntry(
         ? {
             field: normalizeFieldName(entry.field),
             operator: entry.operator,
-            type: matcher,
+            type: os === 'linux' ? 'exact_cased' : matcher,
```

Comment:
Should this check be determined by the input as opposed to hardcoding?

@dasansol92 @paul-tavares is the `matcher` originally based on the `field` input?  So if we have a field `process.executable.caseless` then we'd have the type `exact_caseless`, correct?

Similarly, should we be saving the Linux entries as `process.executable` which translates to `exact_cased`.

Does this sound right?

Reply:
> Should this check be determined by the input as opposed to hardcoding?
> 
> @dasansol92 @paul-tavares is the `matcher` originally based on the `field` input? So if we have a field `process.executable.caseless` then we'd have the type `exact_caseless`, correct?

That is correct

> Similarly, should we be saving the Linux entries as `process.executable` which translates to `exact_cased`.
> 
> Does this sound right?

Yes, we can add an additional check for os where we generate the `matcher` value instead of hardcoding it here! 👍

Reply:
Although, doing that would affect non `wildcard` entry types as well. Just as we do now ( b776305eeb41226d3d0c4f166b7007ffe29837af) with `exact_cased` when the path is not a wildcard.

Reply:
@kevinlog @ferullo @gabriellandau I've added a better linux check in a5d0af1a3c42246a97cfe295619f6175b5313bd4 to output `exact_cased` or `wildcard_cased` for `match` and `wildcard` entries, but not for `match_any`.